internal/scorecard/alpha/scorecard.go

```diff
@@ -39,52 +44,46 @@ type Scorecard struct {
 	bundleConfigMapName string
 	Client              kubernetes.Interface
 	SkipCleanup         bool
+	TestRunner          TestRunner
+}
+
+type PodTestRunner struct {
+	TestConfig Scorecard
+}
+
+type FakePodTestRunner struct {
+	TestConfig Scorecard
```

Comment:
Does that fake test runner need a TestConfig?
```suggestion
```

Reply:
removed TestConfig from FakeTestRunner as it was not used.